.php-cs-fixer.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use PhpCsFixer\Config;
+use PhpCsFixer\Finder;
+
+$finder = Finder::create()
+    ->in(__DIR__)
+    ->name('*.php');
+
+$config = new Config();
+
+$rules = [
+    '@PER-CS2.0'                  => true,
+    'trailing_comma_in_multiline' => ['elements' => ['arguments', 'array_destructuring', 'arrays']], // For PHP 7.4 compatibility
```

Comment:
The PER 2.0 is an extension of the PER 1.0 (the new name of the PSR 12) and its specifications are not compatible on some points with PHP 7.x.

There are 2 solutions for this:
1. Use the PSR 12 right now and move to the PER 2.0 later.
2. Use the PER 2.0 right now with some overriden rules, and remove these overrides later.

Both are OK to me, but I guess that the current solution will be less subject to conflicts and is therefore preferable.